AuctionHouse/Overview.md

```diff
@@ -0,0 +1,101 @@
+* The user can not list the same asset on multiple marketplaces
+* The user has to rely on the marketplaces’ escrow contracts to safely hold their asset.
+
+This is where Auction House shows its power. Its a transaction protocol that allows marketplaces to implement an **escrow-less** sales contract, thus providing ownership of assets to the users.
+
+
+## Creating an Auction House
+The Auction House program can be used to make a marketplace by creating a new **Auction House Instance**. The `Auction House Instance` is a [Program Derived Address (PDA)](https://solanacookbook.com/core-concepts/pdas.html) which is in the keyspace of the Auction House program, ie, the `Instances` are simply public keys owned by the Auction House program.
+
+![](https://i.imgur.com/2HPpM9g.png =300x)
+
+The Instance can be configured in the whatever way the user wants. Here are some important configurable parameters:
+
+* `requireSignOff`: this allows marketplaces to gate which asset can be listed, bought and sold. For every action: list, bid, sale, the instance [authority](https://docs.solana.com/staking/stake-accounts#understanding-account-authorities) needs to sign the transaction
+* `canChangeSalePrice`: `canChangeSalePrice` is only intended to be used with AuctionHouses with `requireSignOff` set to `true`. This allows the Auction House to do complicated order matching to find the best price for the seller.
+* `sellerFeeBasisPoints`: the share of the sale the marketplace takes on all trades
+    
+## Listing and Bidding
+Once the marketplace is active, any trader can post an offer to "sell" an asset on the marketplace. When this happens, the Auction House does two things:
+
+1. Auction House creates a `Sell Order`: in other words, creates a PDA known as the `SellerTradeState` which represents the offer. Trade States are special PDAs that are very cheap in comparison to other PDAs / Accounts. This is because these PDAs only have 1 byte of data, which is the bump of the PDA. All the details of a listings such as list price, amount of tokens, mint account etc, are hashed into the seeds of the PDA and therefore the PDA acts as a "proof of existence" for that listing.
+
+![](https://i.imgur.com/ki27Ds8.png)
+
+2. Auction House also assigns the another PDA: `programAsSigner` PDA as the `Delegate`. Delegates are a feature of the Solana SPL-token program and are discussed in detail [here](https://spl.solana.com/token#authority-delegation). Delegation allows the Auction House Instance to pull assets out of a token account when a sale goes through at a later point.
+
+![](https://i.imgur.com/aIRl7Hb.png)
+
+Similar to the `Sell Order`, when a buyer places a bid, Auction House creates the `BuyerTradeState` PDA representing the bid. The bid amount (native or SPL tokens) then needs to be transferred manually to the `BuyerEscrowAccount` PDA, which holds this amount till the sale goes through.
```

Comment:
There's too much different lingo here. I think we should focus on using:
- To list / A listing: for listing an asset.
- To bid / A bid: for placing a bid on an asset.

In this section, we just need to focus on these two primitive: listing an asset and placing a bid on an asset. You're doing this but the language makes it confusing because we're using words like selling and buying at this point, when actually, buying and selling is a combination of these two primitives with "execute sale" which is explained in the next section.

I'd love to see a more explicit explanation of these two primitives with an example that helps the reader visualise what we're doing.